plugins/cpp/service/include/service/cppservice.h

```diff
@@ -159,6 +159,8 @@ class CppServiceHandler : virtual public LanguageServiceIf
 
     LOCAL_VAR, /*!< This option returns the local variables of a function. */
 
+    RETURN_TYPE, /*!< This option returns the return type of a function. */
```

Comment:
We have to add an icon for this type. Add this line to the `icons.css` file (line: 198):
```js
.icon-Return-type::before
```
The file should look like this:
```js
.icon-Type::before,
.icon-Types::before,
.icon-Return-type::before{
  content : '\e040'
}
```